addons/purchase/report/purchase_quotation_templates.xml

```diff
@@ -37,13 +37,14 @@
                                     <span t-field="order_line.name"/>
                                 </td>
                                 <td class="text-center">
-                                    <span t-field="order_line.date_planned"/>
+                                    <span t-field="order_line.date_planned" t-options="{'widget': 'date', 'format': 'dd/MM/YYYY'}"/>
```

Comment:
Mm, no? Why this change? It depends on the lang/locale of the partner_id this is being sent to. Also doesn't have anything to do with UoMs :thinking:

Reply:
It was a feedback I got
![image](https://github.com/user-attachments/assets/536ab777-187b-42e0-8d5e-18423abcba5b)
But not related indeed :+1:

Reply:
Indeed, it shouldn't show the time. Seems like it could be fixed in a separate bugfix PR because it didn't used to do that, so it should be checked for the earliest version it started happening in